### PR TITLE
Release/4.8.0

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,33 @@ All notable changes to this project will be documented in this file.
...
+
+- ...
+
 ### Changed
 
 - ...
 
+## [4.8.0]
+
+### Added
```
Comment (khssnv) [inline CHANGELOG.md:25]: Should we mention that `fast-unstake` is coming to `[C]`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,12 +5,33 @@ All notable changes to this project will be documented in this file.
...
+
+- [D] Handlebars template to generate weights file
```
Comment (khssnv) [inline CHANGELOG.md:27]: Does not affect `[D]` or `[C]`

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,12 +5,33 @@ All notable changes to this project will be documented in this file.
...
+- [D] Genesis config for `pallet-ddc-staking` to set genesis DDC participants (empty by default) and staking settings
+- [D] Unit tests in `pallet-ddc-staking` for basic staking scenario
```
Comment (khssnv) [inline CHANGELOG.md:29]: Does not affect `[D]` or `[C]`

File: CHANGELOG.md (hunk 1 above)
Comment (MRamanenkau) [reply CHANGELOG.md:25]: `fast-unstake` is included to update to `polkadot-v0.9.30` It doesn't make sense to duplicate it

File: CHANGELOG.md (hunk 3 above)
Comment (MRamanenkau) [reply CHANGELOG.md:29]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (MRamanenkau) [reply CHANGELOG.md:27]: Updated

**Review by yahortsaryk — APPROVED** [review]

**Pull request merged**